test/suites/integration/bsp/submit-proofs.test.ts

```diff
@@ -595,10 +597,6 @@ describeBspNet(
         dummyBspNextChallengeTick < bspTwoNextChallengeTick
           ? dummyBspNextChallengeTick
           : bspTwoNextChallengeTick;
-      const secondBlockToAdvance =
-        dummyBspNextChallengeTick < bspTwoNextChallengeTick
-          ? bspTwoNextChallengeTick
-          : dummyBspNextChallengeTick;
```

Comment:
What's up with this deletion here?

Reply:
both BSPs were responding in the same block. this test was failing because we were expecting the second BSP to respond in a subsequent block to the first one being responded to by the first BSP.

I couldn't see any part of my code that would affect this test.